tensorflow/python/checkpoint/checkpoint_management_test.py

```diff
@@ -97,7 +97,6 @@ def testNameCollision(self):
           self.assertIsNotNone(
               checkpoint_management.latest_checkpoint(traindir))
 
-  @test_util.run_deprecated_v1
```

Comment:
It seems this is still needed. We get an assertion error below otherwise:

self.assertEqual(self.evaluate(v0), 2.0)
AssertionError: 1.0 != 2.0